### PR TITLE
LPS-154230 Company log servlet | 5-25

### PR DESCRIPTION
**Pull request by dantewang**
@yuhai @tinatian

### CODE REVIEW

**Comment by dantewang** [comment]
@tinatian @yuhai This is rebased on top of latest master and https://github.com/liferay-core-infra/liferay-portal/pull/831

File: modules/apps/portal/portal-company-log/src/main/java/com/liferay/portal/company/log/internal/servlet/CompanyLogServletGatekeeper.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Hai Yu
+ */
+@Component(immediate = true, service = {})
+public class CompanyLogServletGatekeeper {
+
+	@Activate
+	protected void activate(ComponentContext componentContext) {
```
Comment (tinatian) [inline modules/apps/portal/portal-company-log/src/main/java/com/liferay/portal/company/log/internal/servlet/CompanyLogServletGatekeeper.java:32]: @yuhai Please follow ModulePortalProfile pattern, see https://issues.liferay.com/browse/LPS-69572 for an example.

FYI @dantewang

**Comment by tinatian** [comment]
@yuhai Please see my comment and resend.

Thanks.
Tina.

**Comment by tinatian** [comment]
ci:close

**Pull request closed without merging**